BasicGates/Tasks.qs

```diff
@@ -111,24 +111,34 @@ namespace Quantum.Kata.BasicGates {
         // ...
     }
 
+    // Task 1.7. Global Phase Change
+    // Inputs:
+    //     1) Pauli operator (μ) represented as Pauli.
+    //     2) Angle alpha, in radians, represented as Double.
+    //     3) A qubit in state β|0⟩ + γ|1⟩.
+    // Goal: Change the state of the qubit to - β|0⟩ - γ|1⟩.
+    operation GlobalPhaseChange (q: Qubit) : Unit is Adj + Ctl {
```

Comment:
Since you changed the signature of this operation, you need to do matching changes in other parts of the code:
* the list of inputs on lines 116-118 of this file
* the signature and list of inputs in `ReferenceImplementation.qs` (and the reference implementation itself, to use 2.0 * PI() instead of the parameter)
* the signature of the matching operation in `BasicGates.ipynb`
* the invocation of the solution and the reference implementation in `Tests.qs` (I think that's what it causing the build break in continuous integration)

Reply:
I am not sure if this is how the function is suppose to look like in Tests.qs

Reply:
Yes, the syntax could be a bit more concise (operation name with (_) as an argument can be written as just operation name, since there is no partial application necessary in this case), but the idea is right